src/connect.rs

```diff
@@ -72,6 +74,14 @@ impl Connector {
             inner: Inner::RustlsTls(http, Arc::new(tls))
         })
     }
+
+    pub(crate) fn local_address(&self, _addr: Option<IpAddr>) {
+        match self.inner {
+        #[cfg(not(feature = "tls"))]
```

Comment:
In order to support even when TLS is enabled, we'll probably need to pass the option during construction instead.